src/usecases/backfill-design-use-case.ts

```diff
@@ -85,6 +85,7 @@ export const backfillDesignUseCase = {
 			await associatedFigmaDesignRepository.upsert({
 				designId: figmaDesignId,
 				associatedWithAri: associateWith.ari,
+				inputUrl: designUrl,
```

Comment:
@akostevich-atlassian should we also modify the associate entity use case?

Reply:
Great catch, Jon. We absolutely should!